src/views/Horas/index.js

```diff
@@ -1,5 +1,10 @@
 import React from 'react'
+import { ActivitiesBarChart } from '../../components/ActivitiesBarChart'
 
 export const Horas = () => {
-  return <div>Horas</div>
+  return (
+    <div>
+      <ActivitiesBarChart />
+    </div>
```

Comment:
Caso tu queira, tu pode passar o componente direto aqui sem a div.

```suggestion
    <ActivitiesBarChart />
```

Ou ainda, substituir a div por um Fragment (componente vazio que só renderiza os filhos - Importante quando se quer retornar dois componentes irmãos sem um pai no HTML)

```suggestion
    <>
      <ActivitiesBarChart />
    </>
```

Sei que isso é temporário e essa substituição talvez bugue o CSS mas comentei mesmo assim pois acho importante tu estar ciente disso.

Reply:
É, assim como na parte dos props padrão, essa parte das páginas é bem provisória, por isso ela está assim mesmo, quando a gente for implementar a issue da página ela vai ser mudada totalmente

Reply:
Perfeito! Comentei mais pra te deixar ciente mesmo.